src/Explorer/Controls/ThroughputInput/ThroughputInputReact.test.tsx

```diff
@@ -0,0 +1,122 @@
+import { shallow } from "enzyme";
```

Comment:
Keep the filename the same as the component that is being tested (ie `ThroughputInputComponent.test.tsx`)